cubist/cubist.py

```diff
@@ -125,19 +139,78 @@ def __init__(self,
         self.n_committees = n_committees
         self.neighbors = neighbors
         self.unbiased = unbiased
+        self.composite = composite
         self.extrapolation = extrapolation
         self.sample = sample
+        self.cv = cv
         self.random_state = random_state
         self.target_label = target_label
         self.verbose = verbose
     
     def _more_tags(self):
         return {"allow_nan": True,
                 "X_types": ["2darray", "string"]}
+    
+    def _validate_model_parameters(self):
```

Comment:
I like this! 

If you wanna be a bit extra, you could collect all errors and then at the end raise a CubistParameterError (or something) alerting the user of all their mistakes.

Reply:
I mean I guess I could, I should also probably make all the functions private that come from private files

Reply:
https://stackoverflow.com/questions/29150132/collect-all-errors-and-raise-reduced-exception
maybe another time...